strands_movebase/strands_movebase_params/costmap_common_params.yaml

```diff
@@ -3,17 +3,32 @@ robot_base_frame: /base_link
 
 map_type: voxel
 
-footprint: [[-0.30,-0.32], # Back side right
-            [-0.52,-0.11], # Back right
-            [-0.52, 0.11], # Back left
-            [-0.30, 0.32], # Back side left
-            [-0.05, 0.31], # Front side left
-            [ 0.05, 0.25], # Thick front left
-            [ 0.15, 0.13], # Thin front left
-            [ 0.20, 0.00], # Front
-            [ 0.15,-0.13], # Thin front right
-            [ 0.05,-0.25], # Thick front right
-            [-0.05,-0.31]] # Front side right
+footprint: [[-0.211159,-0.295000],
```

Comment:
Hmm, yeah, looks good. It's a bit weird that it's not symmetrical.
![footprint](https://cloud.githubusercontent.com/assets/4670159/11438816/b7d5ea82-94f7-11e5-8a4d-300e893dcc30.png)